src/main/java/com/aizistral/infmachine/InfiniteMachine.java

```diff
@@ -280,8 +281,14 @@ public void onSlashCommandInteraction(SlashCommandInteractionEvent event) {
 	    String msg = "<@%s> has been pet.";
 
 	    if (id == 440381346339094539L) {
-		msg = "You should know, that a soul can't be `/pet`\n(CAN'T BE `/PET`!)\n"
-			+ "No matter what machines you wield...";
+			//Added custom bypass of arkadys anti petting code (feel free to remove if you don't agree)
+			if(267067816627273730L == event.getMember().getUser().getIdLong()){
```

Comment:
There's no need to go the long rout of `getMember().getUser()` to get the ID, you can invoke `getUser()` directly on the `event`.